### PR TITLE
Static event page template. No logic, strictly view.

### PR DESCRIPTION
**Pull request by TylerGauntlett**

### CODE REVIEW

File: src/app/index.js [hunk 1]
```diff
@@ -31,6 +31,12 @@ angular.module('app', [
          controller: 'LoginController',
          controllerAs: 'vm'
        })
+        .state('event', {
+          url: '/event',
+          template: require('./components/event/eventView.html'),
+          controller: 'EventController',
+          controllerAs: 'vm'
+        })
```
Comment (Tresky) [inline src/app/index.js]: Could you line this up with the states above it?

File: src/app/styles/index.styl [hunk 2]
```diff
@@ -5,3 +5,4 @@
 // Base Imports
 @import 'typography'
 @import 'variables'
+@import 'global'
```
Comment (Tresky) [inline src/app/styles/index.styl]: Place these in alphabetical order

File: src/public/index.html [hunk 3]
```diff
@@ -14,5 +14,6 @@
     <div nav-bar></div>
     <div ui-view class="site"></div>
     <script src="https://cdn.jsdelivr.net/g/jquery@3.1.1,bootstrap@3.3.7"></script>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```
Comment (Tresky) [inline src/public/index.html]: CSS should be loaded in the <head> of the document. Might even be good to group it in with the normal JSDelivr of whatever stylesheets we loading from them.

File: src/app/components/index.js [hunk 4]
```diff
@@ -1,2 +1,3 @@
 import './home/homeController.js'
+import './event/eventController.js'
 import './login/loginController.js'
```
Comment (Tresky) [inline src/app/components/index.js]: Alphabetical order

File: src/app/components/event/eventController.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+angular.module('app')
+  .controller('EventController', function ($log) {
+    var vm = this
+    vm.test = 'testing'
+    vm.func = () => {
+      $log.log('Function was called')
+    }
+  }
+)
```
Comment (Tresky) [inline src/app/components/event/eventController.js:19]: This file needs to be imported in the `controllers/index.js`

**Review by Tresky — CHANGES_REQUESTED** [review]

File: src/app/index.js (hunk 1 above)
Comment (TylerGauntlett) [reply src/app/index.js]: Done

File: src/app/styles/index.styl (hunk 2 above)
Comment (TylerGauntlett) [reply src/app/styles/index.styl]: Done

File: src/public/index.html (hunk 3 above)
Comment (TylerGauntlett) [reply src/public/index.html]: Move it near the other css include

File: src/app/components/index.js (hunk 4 above)
Comment (TylerGauntlett) [reply src/app/components/index.js]: Done

File: src/app/components/event/eventController.js (hunk 5 above)
Comment (TylerGauntlett) [reply src/app/components/event/eventController.js:19]: Added to `controllers/index.js`

**Review by Tresky — APPROVED** [review]

**Pull request merged**